### DISCUSSION TITLE
Trim whitespace from DOIs on "Insert Citation" screen?

### DISCUSSION THREAD

**Discussion by mikemahoney218** [quarto-dev/quarto-cli#2532] [Feature Requests]
Hi all!

I _believe_ I've got the root cause figured out, though I might be in the wrong repo for RStudio Quarto support:

If you attempt to create a new citation on the "Insert Citation" screen in RStudio using the DOI search option, having any leading or trailing whitespace in your DOI will return "An error occurred while searching for this DOI", with no further explanation. Would it be possible for this menu to trim leading and trailing whitespace "under the hood", as it's likely the result of weird copy-and-paste issues and not an intentional input by the user?

**Comment by dragonstyle** [comment]
I'm able to reproduce this- it would be best logged as an issue in the RStudio repo here: https://github.com/rstudio/rstudio/issues .

Note that if it is bothering you (which you await a fix), you can type the '@' sign and paste a DOI right into the editor and we'll perform the lookup without needing to launch the dialog. That will properly trim the DOI...

**Comment by mikemahoney218** [comment]
Opened https://github.com/rstudio/rstudio/issues/12004
